lib/common/constants.js

```diff
@@ -43,7 +43,8 @@ function constantsFactory (path) {
         },
         Configuration: {
             Files: {
-                Global: process.env.MONORAIL_CONFIG || '/opt/onrack/etc/monorail.json'
+                Global: process.env.MONORAIL_CONFIG || '/opt/monorail/config.json'
+                OnRack: '/opt/onrack/etc/monorail.json'
```

Comment:
Expected '}' to match '{' from line 45 and instead saw 'OnRack'.<br>Expected '}' to match '{' from line 44 and instead saw ':'.<br>Expected '}' to match '{' from line 13 and instead saw '/opt/onrack/etc/monorail.json'.